101-databricks-workspace-with-customer-managed-keys/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,123 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "keyVaultName": {
+      "type": "string",
+      "defaultValue": "[concat(uniqueString(resourceGroup().id))]",
```

Comment:
this must start with a letter, e.g.

[concat('kv', uniqueString(resourceGroup().id))]